qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -5,13 +5,15 @@
 from enum import IntEnum
 from typing import Dict, Union, Optional, Any, Tuple
 
-from qcodes import Instrument, ArrayParameter, Parameter, validators as vals
+from qcodes.instrument.base import Instrument
+import qcodes.utils.validators as vals
+from qcodes.instrument.parameter import Parameter, ArrayParameter, \
+    ParameterWithSetpoints
 
```

Comment:
Would it make sense to add a note to `Qcodes example with Signal Hound USB-SA124B ParameterWithSetpoints` notebook about why this is recommended against `<whatever is there now>`? and a corresponding note in the original driver example notebook?